### PR TITLE
utils: add a convenient python wrapper for swift-api-digester

### PR DESCRIPTION
**Pull request by nkcsgexi**
Based on the same concept of swift-api-dump, this python wrapper
provides framework authors a convenient way to generate baseline or
diagnose ABI/API breakages from an existing toolchain where it has
swift-api-digester installed.

cmake installation part will be added in a separate PR.

rdar://problem/29649661

### CODE REVIEW

**Comment by nkcsgexi** [comment]
@swift-ci please smoke test

File: utils/api_checker/swift-api-checker.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        fatal_error("Need to specify --output")
+
+    if args.action == 'dump':
+        runner = DumpConfig(platform=args.target)
+        runner.run(output=args.output, module=args.module,
+                   swift_ver=args.swift_version, abi=args.abi)
+    elif args.action == 'diagnose':
+        fatal_error('Not implemented')
+    else:
+        fatal_error('Cannot recongize action: ' + args.action)
```
Comment (akyrtzi) [inline utils/api_checker/swift-api-checker.py]: Typo `recongize`

**Review by akyrtzi — APPROVED** [review]
There's a typo, otherwise LGTM.

**Comment by nkcsgexi** [comment]
@swift-ci please smoke test

**Pull request merged**